src/core/api/index.ts

```diff
@@ -730,7 +730,9 @@ class Player extends EventEmitter<PLAYER_EVENT_STRINGS, any> {
         throw new Error(`transport "${transport}" not supported`);
       }
 
-      const transportObj = transportFn(transportOptions);
+      const transportPipelines = transportFn(transportOptions);
+      const customRepresentationFilter =
```

Comment:
``transportOptions && transportOptions.representationFilter`` would achieve roughly the same thing here.

Reply:
Or we could always set at least an empty object in the option_parsers.